src/func.rs

```diff
@@ -579,6 +580,30 @@ impl Func {
         FunctionCall::new(Function::BitOr).arg(expr)
     }
 
+    /// Call `ROUND` function.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use sea_query::tests_cfg::Character::Character;
+    /// use sea_query::{tests_cfg::*, *};
+    ///
+    /// let query = Query::select().expr(Func::round(5.654, 2)).to_owned();
```

Comment:
@IgnisDa could you add example with one argument: (link: https://www.w3resource.com/sqlite/core-functions-round.php)?

Reply:
I was not able to write the function definition for it. Should it be `round.arg(a).arg(b)` or `round.arg(a, b)`?

Reply:
Well, I have some thoughts:

### Have two definitions

```rust
pub fn round<A>(a: A) -> FunctionCall
    where
        A: Into<SimpleExpr>,
        { ... }
        
pub fn round2<A, B>(a: A, b: B) -> FunctionCall
    where
        A: Into<SimpleExpr>,
        B: Into<SimpleExpr>,
        
```

### Have one definition

```rust        
pub fn round2<A, B>(a: A, b: Option<B>) -> FunctionCall
    where
        A: Into<SimpleExpr>,
        B: Into<SimpleExpr>,
        
```

Reply:
I will go with the the first definition. `round` and `round_with_precision`.